src/libs/actions/BankAccounts.ts

```diff
@@ -52,6 +52,13 @@ export {
 export {openPlaidBankAccountSelector, openPlaidBankLogin} from './Plaid';
 export {openOnfidoFlow, answerQuestionsForWallet, verifyIdentity, acceptWalletTerms} from './Wallet';
 
+let bankAccountList: OnyxEntry<BankAccountList>;
+
+Onyx.connectWithoutView({
+    key: ONYXKEYS.BANK_ACCOUNT_LIST,
+    callback: (value) => (bankAccountList = value),
+});
+
```

Comment:
Explanation for this change:

We need to get the last 4 digits of the bank account inside `getReportPreviewMessage`.

This function is used within several functions like createWorkspaceFromIOUPayment, getLastMessageTextForReport, buildOptimisticReportPreview....

Without this usage, we need to add a `bankAccountList` as a new param of `getReportPreviewMessage` and it will require ton of change.